app/controllers/defence_requests_controller.rb

```diff
@@ -30,6 +32,17 @@ def create
     end
   end
 
+  def edit
+  end
+
+  def update
+    if @defence_request.update_attributes(defence_request_params)
+      redirect_to({ action: :index }, notice: t('models.update', model: @defence_request.class))
```

Comment:
As I told Jack use a url helper here rather than a hash as it's quicker:

``` ruby
redirect_to defence_requests_url, notice: t('models.update', model: @defence_request.class)
```